Source/Blazorise/Config.cs

```diff
@@ -27,8 +27,8 @@ public static class Config
         /// <returns></returns>
         public static IServiceCollection AddBlazorise( this IServiceCollection serviceCollection, Action<BlazoriseOptions> configureOptions = null )
         {
-            serviceCollection.Replace( ServiceDescriptor.Transient<IComponentActivator, ComponentActivator>() );
-            serviceCollection.Replace( ServiceDescriptor.Transient<IComponentDisposer, ComponentDisposer>() );
+            serviceCollection.Replace( ServiceDescriptor.Scoped<IComponentActivator, ComponentActivator>() );
+            serviceCollection.Replace( ServiceDescriptor.Scoped<IComponentDisposer, ComponentDisposer>() );
```

Comment:
`serviceCollection.Replace`? Should it?

Reply:
I guess we don't need to have `Replace` as not a lot of people know or use `IComponentActivator`. And if they are using them better to not stay in their ways. OK, I will register it as a regular service.